packages/server/src/sync/service.ts

```diff
@@ -49,6 +52,18 @@ export class SyncService extends Service {
     return result!
   }
 
+  private async syncProvider(clientId: uuid) {
+    const provision = await this.provisions.fetchByClientId(clientId)
+    if (provision) {
+      return provision.providerId
+    }
+
+    const provider = await this.providers.create(clientId, false)
+    await this.provisions.create(clientId, provider.id)
+
+    return provider.id
+  }
```

Comment:
We need to do this now because the provider isn't created when the client is created anymore